app/about/about-page-context.tsx

```diff
@@ -0,0 +1,61 @@
+"use client";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { createContext, useState } from "react";
+
+export enum AboutPageSidebarOption {
+  DATA_EXTRACTION = "DATA_EXTRACTION",
+  FAQ = "FAQ",
+  THE_TEAM = "THE_TEAM",
+}
+
+interface AboutPageSidebarContext {
+  currentSidebarOption: AboutPageSidebarOption | undefined,
+  setCurrentSidebarOption: (route: AboutPageSidebarOption) => void
+}
+
+const initialAboutPageSidebarContext: AboutPageSidebarContext = {
+  currentSidebarOption: undefined,
+  setCurrentSidebarOption: () => {}
+}
+
+export const aboutPageSidebarContext = createContext<AboutPageSidebarContext>(initialAboutPageSidebarContext);
+
+const AboutPageSidebarProvider = ({
+  children,
+}: {
+  children: React.ReactNode;
+}): React.ReactNode => {
+  const [currentSidebarOption, setCurrentSidebarOption] = useState<AboutPageSidebarOption | undefined>(undefined);
+  
+  return (
+    <aboutPageSidebarContext.Provider value={{
+      currentSidebarOption,
+      setCurrentSidebarOption
+    }}>
+      {children}
+    </aboutPageSidebarContext.Provider>
+  );
+};
+
+export const AboutPageProvider = ({
+  children,
+}: {
+  children: React.ReactNode;
+}): React.ReactNode => {
+  const queryClient = new QueryClient({
+    defaultOptions: {
+      queries: {
+        staleTime: 86400000, // One day
```

Comment:
I see the value in this but our stale time could be much longer also/could be infinte (the cache time and stale time only matter for current session afaik - might be worng but this is my understanding so please correct me if not) Our team data will not change often enough to use caching at all. infact the page will not be accessed often enough either so we could just call it everytime (maybe to prevent spam we can add stale time) anyway all this is just discussion and I feel like I have come to a point where a 1 day long stale time and cache time is good for preventing someone from spamming our servers accidentally even (maybe overkill too ahahaha)